spec/factories/extra_marks.rb

```diff
@@ -11,4 +11,10 @@
     unit { 'points' }
     extra_mark { 1 }
   end
+  factory :extra_mark_percentage_of_score, class: 'ExtraMark' do
```

Comment:
This is okay, but please refactor all three factories to use a common parent factory and then has three child factories, one for each type. (You'll need to research how to create parent-child relationships with factories using `FactoryBot`.)